### PR TITLE
feat(editors/substation): redirect LNode's on clone

### LINKED ISSUE #1053: Redirect `LNode` on cloning `Bay`, `VoltageLevel` and `Substation`
As a user of OpenSCD I have to design the same bay with little differences over and over again and want OpenSCD to give me an efficient process to do so.

**Requirements**
1. extend the current clone behavior to incorporate copying of `LNode` elements
>NOTE:  as `LNode` reference is unique within the whole SCL a plane copy is not possible instead we can replace with `LNode` to the same data in another `IED`
- take over all `LNode` elements within the to be copied entity (`Bay`, `VoltageLevel` or `Substation`)
- do no change any `LNode` element that has `iedName=None`
- for each `LNode` where `iedName` is not `None` get the IED to be replaced with from the UI (see mock)
   - If `none` is selected (see Mock), remove all `LNode`
   - If not `none` is selected e.g.`OhterIED2` check whether the reference is valid in the new IED and replace the `iedName` attribute in the `LNode` in case the reference is valid. Otherwise, remove the `LNode`. Valid means that the reference as a combination of IED LDevice and logical node is pointing to a valid reference
- for each IED print on select all per default `none`
- if there are no `LNode` elements in the to be copied entity or all `LNode` are `iedName=None` to no show the UI

Mockup:
grafik

### PR DESCRIPTION
**Pull request by JakobVogelsang**
Close #1053

### CODE REVIEW

File: test/unit/editors/substation/redirectionUI.test.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  }
+
+  redirectUI
+    .querySelector<HTMLElement>('mwc-button[icon="content_copy"]')
+    ?.click();
+  await redirectUI.updateComplete;
+}
+
+describe('Clone Redirection UI', () => {
+  let element: BayEditor;
```
Comment (dlabordus) [inline test/unit/editors/substation/redirectionUI.test.ts:55]: Don't we want these snapshot tests in the different editors and not only with the BayEditor?

File: test/unit/editors/substation/__snapshots__/redirectionUI.test.snap.js [hunk 2]
```diff
@@ -0,0 +1,195 @@
+/* @web/test-runner snapshot v1 */
```
Comment (dlabordus) [inline test/unit/editors/substation/__snapshots__/redirectionUI.test.snap.js:1]: It contains 2 snapshots, but I can only find one case having a snapshot.

Comment (JakobVogelsang) [reply test/unit/editors/substation/__snapshots__/redirectionUI.test.snap.js:1]: Thx. That was a test rename that added the second snapshot. In those cases, a new snapshot must be generated. I fixed it

File: test/unit/editors/substation/redirectionUI.test.ts (hunk 1 above)
Comment (JakobVogelsang) [reply test/unit/editors/substation/redirectionUI.test.ts]: Nice seen. I did trigger the redirect UI from all the three editors. Thx

**Review by dlabordus — APPROVED** [review]

**Pull request merged**